front/components/members/RenderInvitations.tsx

```diff
@@ -0,0 +1,80 @@
+import { Avatar, ChevronRightIcon, Chip, Icon } from "@dust-tt/sparkle";
+import type { MembershipInvitationType } from "@dust-tt/types";
+
+import { displayRole, ROLES_DATA } from "@app/components/members/Roles";
+
+export function RenderInvitations({
```

Comment:
No need for render in the name of the component. We can go with `InvitationsList` or something similar.